### PR TITLE
Add support for moving and deleting keys

### PR DESCRIPTION
**Pull request by aveenismail**

### CODE REVIEW

**Review by qpernil — APPROVED** [review]

File: tool/yubico-piv-tool.c [hunk 1]
```diff
@@ -2588,6 +2612,23 @@ int main(int argc, char *argv[]) {
...
+      case action_arg_moveMINUS_key: {
+        int from_slot = get_slot_hex(args_info.slot_arg);
+        int to_slot = get_slot_hex((enum enum_slot) args_info.to_slot_arg);
+        if (move_key(state, from_slot, to_slot) == false) {
+          ret = EXIT_FAILURE;
+        } else {
+          fprintf(stderr, "Successfully moved key.\n");
+        }
+        break;
+      }
```
Comment (dainnilsson) [inline tool/yubico-piv-tool.c:2624]: Should this case really have curly braces around it?

Comment (aveenismail) [reply tool/yubico-piv-tool.c:2624]: It starts with an assignment. Although I am thinking of moving the two get_slot_hex() calls to move_key() call but wouldn't that make the line too long?

Comment (dainnilsson) [reply tool/yubico-piv-tool.c:2624]: Ah, that makes sense.

**Review by qpernil — COMMENTED** [review]
Some comments left in the code

**Pull request merged**